WordPress/Classes/ViewRelated/Reader/ReaderPostMenu.swift

```diff
@@ -212,7 +212,7 @@ open class ReaderPostMenu {
                                             object: nil,
                                             userInfo: [ReaderNotificationKeys.post: post])
         }, failure: { _ in
-            ReaderHelpers.dispatchToggleSeenError(post: post)
+            ReaderHelpers.dispatchToggleSeenMessage(post: post, success: false)
```

Comment:
`ReaderPostMenu` isn't used anymore. This change just complies with the method change.